src/coreclr/System.Private.CoreLib/src/System/Threading/Overlapped.cs

```diff
@@ -52,8 +52,6 @@ internal static void PerformIOCompletionCallback(uint errorCode, uint numBytes,
                     ExecutionContext.RunInternal(helper._executionContext, IOCompletionCallback_Context_Delegate, helper);
                 }
 
```

Comment:
The whole `_IOCompletionCallback` type can be deleted. Also delete `IOCB_HELPER` in VM that is referencing it from the VM.